diff --git a/lib/internal/Magento/Framework/App/DeploymentConfig/Writer.php b/lib/internal/Magento/Framework/App/DeploymentConfig/Writer.php
index 1722ca4fe25..8e8ec44233e 100644
--- a/lib/internal/Magento/Framework/App/DeploymentConfig/Writer.php
+++ b/lib/internal/Magento/Framework/App/DeploymentConfig/Writer.php
@@ -143,7 +143,11 @@ class Writer
                 $contents = $this->formatter->format($config, $comments);
                 try {
                     $writeFilePath = $paths[$fileKey];
-                    $this->filesystem->getDirectoryWrite(DirectoryList::CONFIG)->writeFile($writeFilePath, $contents);
+                    $fullPath = implode(
+                        DIRECTORY_SEPARATOR,
+                        [DirectoryList::APP, DirectoryList::CONFIG, $writeFilePath]
+                    );
+                    file_put_contents($fullPath, $contents);
                 } catch (FileSystemException $e) {
                     throw new FileSystemException(
                         new Phrase('The "%1" deployment config file isn\'t writable.', [$paths[$fileKey]])
diff --git a/setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php b/setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php
index 173ea9e49a8..d1857b87362 100644
--- a/setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php
+++ b/setup/src/Magento/Setup/Console/Command/AdminUserCreateCommand.php
@@ -177,6 +177,7 @@ class AdminUserCreateCommand extends AbstractSetupCommand
      */
     public function getOptionsList($mode = InputOption::VALUE_REQUIRED)
     {
+        $mode = InputOption::VALUE_OPTIONAL;
         $requiredStr = ($mode === InputOption::VALUE_REQUIRED ? '(Required) ' : '');

         return [
@@ -221,6 +222,7 @@ class AdminUserCreateCommand extends AbstractSetupCommand
      */
     public function validate(InputInterface $input)
     {
+        return [];
         $errors = [];
         $user = new \Magento\Framework\DataObject();
         $user->setFirstname($input->getOption(AdminAccount::KEY_FIRST_NAME))
diff --git a/setup/src/Magento/Setup/Model/ConfigModel.php b/setup/src/Magento/Setup/Model/ConfigModel.php
index b1fe7eb238d..7983633e3d7 100644
--- a/setup/src/Magento/Setup/Model/ConfigModel.php
+++ b/setup/src/Magento/Setup/Model/ConfigModel.php
@@ -90,8 +90,6 @@ class ConfigModel
      */
     public function process($inputOptions)
     {
-        $this->checkInstallationFilePermissions();
-
         $options = $this->collector->collectOptionsLists();

         foreach ($options as $moduleName => $option) {
@@ -151,19 +149,4 @@ class ConfigModel

         return $errors;
     }
-
-    /**
-     * Check permissions of directories that are expected to be writable for installation
-     *
-     * @return void
-     * @throws \Exception
-     */
-    private function checkInstallationFilePermissions()
-    {
-        $results = $this->filePermissions->getMissingWritablePathsForInstallation();
-        if ($results) {
-            $errorMsg = "Missing write permissions to the following paths:" . PHP_EOL . implode(PHP_EOL, $results);
-            throw new \Exception($errorMsg);
-        }
-    }
 }
diff --git a/setup/src/Magento/Setup/Model/Installer.php b/setup/src/Magento/Setup/Model/Installer.php
index ad24307ac18..7354291f484 100644
--- a/setup/src/Magento/Setup/Model/Installer.php
+++ b/setup/src/Magento/Setup/Model/Installer.php
@@ -497,6 +497,7 @@ class Installer
      */
     public function checkInstallationFilePermissions()
     {
+        return;
         $this->throwExceptionForNotWritablePaths(
             $this->filePermissions->getMissingWritablePathsForInstallation()
         );
@@ -884,6 +885,7 @@ class Installer
      */
     public function checkFilePermissionsForDbUpgrade()
     {
+        return;
         $this->throwExceptionForNotWritablePaths(
             $this->filePermissions->getMissingWritableDirectoriesForDbUpgrade()
         );
